dev_guide/secrets.adoc

```diff
@@ -80,6 +108,30 @@ $ oc create -f <filename>
 ----
 ====
 
+[[types-of-secrets]]
+=== Types of Secrets
+
+The value in the `type` field is a hint at the structure of the secret's key names and values.
```

Comment:
Maybe instead of saying it's a "hint" maybe say that it indicates the type of structure?

Reply:
Ooops!